pkg/vmlinux/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
```

Comment:
vmlinux.h is not compiled by CGO in this case, right?

Reply:
Right.

I thought that any potential external git repository that import some gadgets from `pkg/gadgets/...` would need to import `_ "github.com/kinvolk/inspektor-gadget/pkg/vmlinux"` as well so it is vendored... But I am not sure it works like that: I have not tested it and I didn't find the documentation on that.